schema/codelists/documentType.csv

```diff
@@ -1,15 +1,15 @@
 award,awardNotice,Award notice,"Notice published by the buyer or procuring entity giving details of the contract award. This notice is published after a contract has been awarded but before it has been signed: for example, for the purposes of the standstill period. Once the contract has been signed, a contract signature notice is typically published.",,
 contract,contractNotice,Contract signature notice,"Notice published by the buyer or procuring entity giving details of the contract being concluded (for example, signed). This notice is called a contract signature notice, a result notice, a contract award notice (for example, in the EU), or a contract notice (for example, in Australia).",,
 implementation,completionCertificate,Completion certificate,A document issued by a relevant authority certifying that the work specified in the contract is complete to a certain level of quality. Completion certificates might only be relevant for particular kinds of contracting processes.,,
-planning,procurementPlan,Procurement plan,Documentation that sets out the basis for this particular contracting process.,,
+planning,procurementPlan,Procurement plan,Documentation that sets out the basis for the future contracting process.,,
 tender,biddingDocuments,Bidding documents,"Documentation for potential suppliers, describing the goals of the contract (e.g. goods and services to be procured), and the bidding process.",,
 tender,technicalSpecifications,Technical specifications,"A document that describes the characteristics of goods or services to be procured, including quality, performance, safety and dimensions, or the processes and methods for their production or provision. Or, a document that addresses terminology, symbols, packaging, marking or labelling requirements, as they apply to a good or service.",,
-tender,exclusionGrounds,Exclusion grounds,"Documentation that describes the criteria regarding the situation of a tenderer that can lead to its exclusion from the contracting process. For example: criminal convictions, bankruptcy, presence on a blacklist or failure to pay taxes.",,
-tender,selectionCriteria,Selection criteria,Documentation that describes the minimum requirements for potential suppliers to participate in the contracting process.,,
+tender,exclusionGrounds,Exclusion grounds,"Documentation that describes the criteria regarding the situation of a tenderer that can lead to its exclusion from the future contracting process. For example: criminal convictions, bankruptcy, presence on a blacklist or failure to pay taxes.",,
+tender,selectionCriteria,Selection criteria,Documentation that describes the minimum requirements for potential suppliers to participate in the future contracting process.,,
```

Comment:
Sorry, this was my mistake. (I got mentally sidetracked along the lines of "participating in a process", "requesting to participate in a process", "requesting to participate in a future process?" etc., see https://github.com/open-contracting/standard/pull/1464#discussion_r774787527.) 

Replacing process with activities - I rather wouldn't. First, I think we should try to stick to the defined term, if we can. Second, I wouldn't do anything special for these EU PINs, because it's not all the EU PINs, but just for the "PINs used as call for competition", which the eForms classifies under Competition forms, so they wouldn't start a planning process but a contracting process. (Regrettably, this information is also in "PIN used to shorten time limits", which is classified as a planning notice, but the scenario is quite narrow, jurisdiction-specific and - I think - not hugely used, so I wouldn't change our wording just because of it.)

Reply:
Done in https://github.com/open-contracting/standard/pull/1513/commits/93b050c385e810bad6fb03b1ccc04fd19a8276ce